### PR TITLE
Graph Repo Updates

### PR DESCRIPTION
**Pull request by codenamejason**
##### Description

Graph repo updates for merging the projects under one repo.

##### Refers/Fixes

Repo refactoring for graph

### CODE REVIEW

**Comment by DanieleSalatti** [comment]
`fleek/build`, `gegitcoin` and `rmgitcoin` still failing...

**Comment by thelostone-mc** [comment]
Rebase against the main repo once and it should pass

File: .github/workflows/graph-round.yml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        with:
+          node-version: "16"
+          cache: "yarn"
+          cache-dependency-path: packages/graph/round/yarn.lock
+
+      - name: Install Dependencies for Round Manager
+        run: |
+          yarn install
+          cd packages/graph/round
+          yarn install
```
Comment (thelostone-mc) [inline .github/workflows/graph-round.yml:27]: ^ these would fail cause it relies on pnpm

**Comment by melissa-neira** [comment]
defer to engineering 🕺

**Review by thelostone-mc — APPROVED** [review]

**Pull request merged**